modules/index.d.ts

```diff
@@ -276,21 +277,22 @@ export type TreeSelectWidgetProps = BaseWidgetProps & TreeSelectFieldSettings;
 export type RangeSliderWidgetProps = RangeWidgetProps & NumberFieldSettings;
 
 export interface BaseWidget {
-  customProps?: AnyObject,
-  type: string,
-  jsType?: string,
-  factory: Factory<WidgetProps>,
-  valueSrc?: ValueSource,
-  valuePlaceholder?: string,
-  valueLabel?: string,
-  fullWidth?: boolean,
-  formatValue: FormatValue,
-  sqlFormatValue: SqlFormatValue,
-  mongoFormatValue?: MongoFormatValue,
-  elasticSearchFormatValue?: ElasticSearchFormatValue,
-  hideOperator?: boolean,
+  customProps?: AnyObject;
+  type: string;
+  jsType?: string;
+  factory: Factory<WidgetProps>;
+  valueSrc?: ValueSource;
+  valuePlaceholder?: string;
+  valueLabel?: string;
+  fullWidth?: boolean;
+  formatValue: FormatValue;
+  sqlFormatValue: SqlFormatValue;
+  mongoFormatValue?: MongoFormatValue;
+  elasticSearchFormatValue?: ElasticSearchFormatValue;
+  hideOperator?: boolean;
+  jsonLogic?: string | JsonLogicFormatValue;
```

Comment:
String is not supported, only function.
https://github.com/ukrbublik/react-awesome-query-builder/blob/2666f3c9e8d1ee1ab32cf27d94eb05f115b21ccc/modules/export/jsonLogic.js#L254

Reply:
Changed to Function type